### PR TITLE
Adding environment testing and short version to nb adding instructions

### PR DESCRIPTION
**Pull request by rmshkv**
I added a step telling people to install the cupid-analysis environment and test there. I also noted where people can stop if they don't want to fully run the notebook with CUPiD themselves yet, so they can pass it along to us at that point instead.

Addresses https://github.com/NCAR/CUPiD/issues/70.

### CODE REVIEW

**Review by mnlevy1981 — CHANGES_REQUESTED** [review]

File: docs/addingnotebookstocollection.md [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
+3. Tag this cell as `parameters`. This means that when the notebook is executed by `CUPiD`, a new cell will be inserted just below this one with all of the parameters you specify in `config.yml` (see step 4). To tag it, in Jupyter Lab, click on the cell and click the button with two gears in the top right ("Property Inspector"). Open "Common Tools." There, you can see a section called "Cell Tags." Click "Add Tag," and add one called `parameters` (exactly as written). **If you don't want to fully set up CUPiD, stop here and we can integrate the notebook into a CUPiD workflow from here.**
+---
+4. Move your new diagnostic notebook to the folder called `examples/nblibrary`.
 
-4. Open `config.yml`. First, add your new notebook (as its name, minus the `.ipynb`) to the list of notebooks that will be computed (`compute_notebooks`). The notebooks will be executed in the order they are listed here. For example:
+5. Open `config.yml`. First, add your new notebook (as its name, minus the `.ipynb`) to the list of notebooks that will be computed (`compute_notebooks`). The notebooks will be executed in the order they are listed here. For example:
```
[the remark this answers is not in the document]
Comment (rmshkv) [reply docs/addingnotebookstocollection.md:19]: Updated this!

File: docs/addingnotebookstocollection.md [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
 	    data_path = "path/to/data"
 	    dates = {"start_date" = "01/01/01",
 	    			    "end_date" = "01/01/02"}
 
-3. Tag this cell as `parameters`. This means that when the notebook is executed by `cupid`, a new cell will be inserted just below this one with all of the parameters you specify in `config.yml` (see step 4). To tag it, in Jupyter Lab, click on the cell and click the button with two gears in the top right ("Property Inspector"). Open "Common Tools." There, you can see a section called "Cell Tags." Click "Add Tag," and add one called `parameters` (exactly as written).
+3. Tag this cell as `parameters`. This means that when the notebook is executed by `CUPiD`, a new cell will be inserted just below this one with all of the parameters you specify in `config.yml` (see step 4). To tag it, in Jupyter Lab, click on the cell and click the button with two gears in the top right ("Property Inspector"). Open "Common Tools." There, you can see a section called "Cell Tags." Click "Add Tag," and add one called `parameters` (exactly as written). **If you don't want to fully set up CUPiD, stop here and we can integrate the notebook into a CUPiD workflow from here.**
```
[the remark this answers is not in the document]
Comment (rmshkv) [reply docs/addingnotebookstocollection.md]: Fixed

File: docs/addingnotebookstocollection.md [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
+---
```
[the remark this answers is not in the document]
Comment (rmshkv) [reply docs/addingnotebookstocollection.md:14]: Yeah, I added it for the visual separation. I added a bit of text below it explaining the division again - but if you think that's unclear, feel free to change it to how you prefer!

File: docs/addingnotebookstocollection.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mnlevy1981) [reply docs/addingnotebookstocollection.md:19]: There are a few more, you can use `grep` to find them all:

```
$ grep -nP '\t' addingnotebookstocollection.md
6:	    sname = "run_name"
7:	    data_path = "path/to/data"
8:	    dates = {"start_date" = "01/01/01",
9:	    			    "end_date" = "01/01/02"}
15:		your_new_nb_name:
16:			    parameter_groups:
17:				    none:
21:	If you just want the notebook run once on one set of parameters, keep the `parameter_groups: none:` notation as above. If you want the notebook executed multiple times with different parameter sets, the notation would look like this:
23:		your_new_nb_name:
```

[the remark this answers is not in the document]
Comment (rmshkv) [reply docs/addingnotebookstocollection.md:19]: Oops, I think that's all of them? Was editing in the online editor so it was a bit tricky to catch

[the remark this answers is not in the document]
Comment (mnlevy1981) [reply docs/addingnotebookstocollection.md:19]: Yup, looks like that's all of them. Thanks!

**Review by mnlevy1981 — APPROVED** [review]
Looks great!

**Pull request merged**